libs/sdk-model/src/insight/tests/sanitization.test.ts

```diff
@@ -81,4 +82,33 @@ describe("insightSanitize", () => {
         expect(resultTotals1).toEqual([grandTotal]);
         expect(resultTotals2).toEqual([grandTotal]);
     });
+
+    it("should remove duplicate drillable bucket attribute items", () => {
+        const targetDrillUri = "/gdc/md/heo9nbbna28ol3jnai0ut79tjer5cqdn/obj/1091";
+        const departmentUri = "/gdc/md/heo9nbbna28ol3jnai0ut79tjer5cqdn/obj/1103";
+        const insight = newInsightDefinition("foo", (m) =>
+            m.buckets([
+                { localIdentifier: "measures", items: [m1, m2] },
+                {
+                    localIdentifier: "attribute",
+                    items: [
+                        modifyAttribute(a1, (a) => a.displayForm(uriRef(targetDrillUri))),
+                        modifyAttribute(a2, (a) => a.displayForm(uriRef(targetDrillUri))),
+                        modifyAttribute(a3, (a) => a.displayForm(uriRef(departmentUri))),
+                    ],
```

Comment:
why do you need to do this? seems like somewhere, we are not treating refs opaquely (which will lead to trouble down the road with tiger). why are idRefs not sufficient?

Reply:
I just used existing attributes, not sure why the modifications were there https://github.com/gooddata/gooddata-ui-sdk/pull/585/commits/0b29e179841472e82c59a193e7b9578fd9348797